nemo/collections/tts/models/magpietts.py

```diff
@@ -2037,7 +2078,11 @@ def infer_batch(
         maskgit_fixed_schedule=None,
         maskgit_dynamic_cfg_scale=False,
         maskgit_sampling_type=None,
+        ignore_finished_sentence_tracking=False,
+        eos_detection_method="argmax_or_multinomial_any",
     ):
+
```

Comment:
We should add a docstring to infer_batch since there any many arguments in this function now.